config/plugins/webhooks/gtn/script.js

```diff
@@ -142,6 +142,26 @@
                         }
                         removeOverlay();
                     });
+
+                    var gtn_workflows = $("#gtn-embed").contents().find("span[data-workflow]");
```

Comment:
```suggestion
                    const gtn_workflows = $("#gtn-embed").contents().find("span[data-workflow]");
```

Reply:
I don't remember resolving this but 
```suggestion
                    let gtn_workflows = document.getElementById("gtn-embed").contentDocument.querySelectorAll("span[data-workflow]")
```